server/poll/poll.go

```diff
@@ -39,10 +41,12 @@ type AnswerOption struct {
 	Voter  []string
 }
 
-// Settings stores possible settings for a poll
+// Settings stores possible settings for a poll.
+// When adding new settings, to avoid failures during atomic transactions for KV Store,
+// either specify omitempty or set initial values during the upgrade.
 type Settings struct {
 	Anonymous        bool
-	AnonymousCreator bool
+	AnonymousCreator bool `json:",omitempty"`
```

Comment:
Should we also specify the JSON field name here?

Reply:
Thanks @hanzei , but I'll leave it as is for now.  Because if we specify the field name, it must be `json:"AnonymousCreator,omitempty"` for compatibility, but it looks little weird.  And features works fine without specifying the JSON field name.